core/templates/dev/head/components/summary_tile/ExplorationSummaryTileMobileDirective.js

```diff
@@ -0,0 +1,141 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for an exploration summary tile.
+ */
+
+oppia.directive('explorationSummaryTileMobile', [function() {
```

Comment:
Well, for the mobile preview card the dimensions are different, the statistics part at the bottom isn't shown and there is a "play exploration" button right below the objectives section. So I thought it'd be a lot cleaner if we had a separate directive for the mobile version.